workers/unity/Packages/com.improbable.gdk.tools/LocalLaunch.cs

```diff
@@ -204,7 +204,7 @@ public static void LaunchLocalDeployment()
                 command = "osascript";
                 commandArgs = $@"-e 'tell application ""Terminal""
                                      activate
-                                     do script ""cd {SpatialProjectRootDir} && {Common.SpatialBinary} {command}""
+                                     do script ""cd {SpatialProjectRootDir} && {Common.SpatialBinary} local launch""
```

Comment:
Could do the same thing here as in the other location? To re-include the prior commandArgs definition?